internal/acctest/serversettings/server_settings_test.go

```diff
@@ -252,6 +269,14 @@ func testAccCheckExpectedServerSettingsAttributes(config serverSettingsResourceM
 			}
 		}
 
+		if config.captchaSettings != nil {
+			err = acctest.TestAttributesMatchString(resourceType, nil, "site_key",
```

Comment:
Does PF even return this? I assumed it would be empty since it's a "key" of some sort

Reply:
It returns siteKey and encryptedSecretKey, just not secretKey - like the password behavior.